### ISSUE TITLE
Multiple UPE - Remove "new checkout experience" setting if it is no longer needed

### ISSUE THREAD

**Issue by harriswong** [Automattic/woocommerce-payments#4724]
### Description
This is part of the Multiple UPE project (https://github.com/Automattic/woocommerce-payments/pull/3968).

The "new checkout experience" setting is used because UPE consolidates all payment methods into 1. But the multiple UPE branch splits payment methods back to individual ones so the setting is no longer needed. Remove this setting. 

### Acceptance criteria
- Remove the "new checkout experience"

### Additional context
Check paJDYF-5dK-p2#comment-13768 for additional context.

**Comment by gpressutto5** [comment]
Removing the "new checkout experience" setting (`_wcpay_feature_upe`) also includes removing any banner, button, or message related to disabled UPE or referencing the "Early access" to this feature (besides the "Early Access" badge) because they are all tied to this flag. After the changes, wcpay will behave as if the "new checkout experience" was always enabled, minus the options to disable it.

<table>
  <tr>
    <th>Banner</th>
    <th>Wizard</th>
  </tr>
  <tr>
    <td> The list of payment methods checkboxes will always be displayed instead of this banner. <br /> <img src="https://user-images.githubusercontent.com/12385501/193353407-56ce2f1c-eab5-49f9-864e-39bffd6db007.png" /> </td>
    <td> Because there is no banner, there's no button to redirect to this page, but I think that's fine now that we will show all payment methods on the previous page.<br /><img src="https://user-images.githubusercontent.com/12385501/193353584-71f026a9-c997-4ab8-8d75-a96980ba6eb0.png" /> </td>
  </tr>
  <tr>
    <th colspan="2"> Disable/feedback </th>
  </tr>
  <tr>
    <td colspan="2"> The disable button should be removed. Should we keep the "Provide feedback" button?<br /><img src="https://user-images.githubusercontent.com/12385501/193353856-111feaaf-ce79-4e00-a953-08ca4a5116c3.png" /> </td>
  </tr>
</table>


I talked to @FangedParakeet, and it looks like there's no need for the disable button anyone since there's no functionality broken by UPE anymore, and disabling the individual methods achieves the same goal as disabling UPE.

> The option to disable is basically included because there has been functionality that has been broken by the UPE, where disabling is a fallback to return the checkout to a stable state. Whereas now--since the card gateway uses non-UPE Stripe elements--disabling basically only removes the newer payment methods, which didn't even exist before. In that sense, there's not really anything to be gained by disabling the UPE.

So, technically, we could "release" UPE and leave the Early Access. But from a product perspective, do we want to do this? Could you help us with this @elizaan36?

**Comment by elizaan36** [comment]
> The list of payment methods checkboxes will always be displayed instead of this banner.

This makes sense to me.

> Wizard

Is there definitely no Inbox note or task directing to the wizard? I remember there was one previously but it may have been removed already. I think we should keep the wizard for future setup improvements, but just remove the "New"/"Early access" language.

> it looks like there's no need for the disable button anyone since there's no functionality broken by UPE anymore, and disabling the individual methods achieves the same goal as disabling UPE.

For things like getting WooPay working on the store, I believe we still need the ability to disable the UPE at this level. Or is it the case that it won't be necessary with the upcoming split UPE work?

> So, technically, we could "release" UPE and leave the Early Access. But from a product perspective, do we want to do this? Could you help us with this @elizaan36?

I definitely think it makes sense to remove the banner and any "Early access" language. cc @pierorocca in case you have thoughts on this as well.

**Comment by gpressutto5** [comment]
Thank you, @elizaan36!

> For things like getting WooPay working on the store, I believe we still need the ability to disable the UPE at this level. Or is it the case that it won't be necessary with the upcoming split UPE work?

That won't be the case anymore. With UPE-split, we will use the existing Credit Card element even when UPE is enabled, so there's no use in disabling UPE.

**Comment by gpressutto5** [comment]
@elizaan36 Mayisha noticed this:

> In this implementation the methods are listed in the settings from the beginning but in disabled state. Shouldn't we still let the merchant's know that they can enable some additional payment methods?

It means both existing users and new installations will have new payment methods available for them, but they won't know until they access the Payments settings page. I thought about adding an Onboarding Task that would lead them to the wizard we mentioned above. It would look like this:

![image](https://user-images.githubusercontent.com/12385501/199323359-864343ae-b03f-4e43-bbbc-7da353ee2113.png)

We would need to reword it to differentiate it from the "Set up additional payment methods" above, which redirects the user to a page where they can browse recommended payment methods from the WC Extensions Store.
It would behave like the "Review Shipping Options", which gets completed whenever the user clicks them and is redirected to the settings page. What do you think about this solution?

One problem I can think of this is we already have the "Set up WooCommerce Payments" task that is marked as completed for any user that has already set up wcpay, but new users might use this task to set up UPE on the settings page, making the new task not beneficial for them. A way to solve this would be to hide the new task in case any of the UPE methods is enabled.

-----

Another way to solve this issue without using tasks would be to add a dismissible notice with a button to review the additional payment methods. This option would be better for existing users but not suitable for new installations, IMO.

**Comment by elizaan36** [comment]
hey @gpressutto5 Good thinking here.

It would be quite confusing for merchants to see two almost identical tasks about additional payment methods after they enabled WC Payments. We should use the existing "Set up additional payment options" task for the UPE/local payment methods in addition to the payment methods in the marketplace. Merchants don't think of these differently so we should try to simplify and make it easier to setup what's best for your store. 

This flow hasn't been built yet but we need to take a more holistic approach to payment method setup so that the tasks and workflows aren't quite so fragmented. @rogermattic is working on a flow that could be triggered from the existing task to make it more useful for new merchants so let's hold off on implementing any new task or notice for now.

**Comment by elizaan36** [comment]
@gpressutto5 Here's an option to use the existing "Set up additional payments" task. 

What if we added in a section for "Local payment methods" to the existing task screen? 
<img width="736" alt="Screen Shot 2022-11-09 at 11 35 43 AM" src="https://user-images.githubusercontent.com/5121465/200901527-93280561-4b61-46d3-a9a0-6b294c5c337c.png">

The link to "Get started" would launch the vertical stepper.
<img width="672" alt="Screen Shot 2022-11-09 at 11 40 45 AM" src="https://user-images.githubusercontent.com/5121465/200901883-d7e97467-3456-4fa0-b3bc-e69c4fd8fdb0.png">

The task will not be completed however until one of the methods from the "Additional payment options" list is enabled.

cc @rogermattic

**Comment by elizaan36** [comment]
Now that I'm looking at the vertical stepper again, we'll need to remove the Credit card/debit card option since this is no longer the UPE package.

**Comment by FangedParakeet** [comment]
We have decided to initially rollout the split-UPE in parallel with the "new checkout experience", so that we can use a server side flag to incrementally release the split-UPE and gradually release the "new checkout experience" as and when we are assured that the split-UPE is stable in production. For this reason, this task will have to be postponed until that distant time when the split-UPE has truly earned the right to be a complete successor to the reigning "new checkout experience".

Please see #4671 and #5271 for more context.

**Comment by frosso** [comment]
Will be superseeded by the Deferred Intent UPE rollout. Closing.
